Add luigi bootstrap dom element option (#707)
SAP/luigi: docs/luigi-ux-features.md

# Luigi UX features

### Rendering of Luigi application in the DOM

> By default, the Luigi content (top navigation, left navigation and content iframed area) are rendered in the body tag of your Luigi Core application. In this case, the Luigi content is taking the whole space from your browser window.

Reviewer: ```suggestion
By default, the Luigi content, including the top navigation, the left navigation, and the content iframed area, are rendered in the body tag of your Luigi Core application. As a result, the Luigi content takes the whole space from your browser window.
```

> There is also the possibility to render the Luigi content in any other html container. For example in case you want to add a header or a footer on top of the Luigi content. All you have to do is add the html custom attribute `luigi-app-root` to the html tag where you want to render the Luigi content.

Reviewer: ```suggestion
However, you can render the Luigi content in any other HTML container. It can be useful if you want to add a header or a footer on top of the Luigi content. To use this feature, add the `luigi-app-root` custom HTML attribute to the HTML tag in which you want to render the Luigi content.
```

> >**NOTE:** When rendering the Luigi content in a custom container, please consider the container is positioned relatively when applying your own css styles. Additionally, the height of the Luigi custom container should be set either in _px_ or _vh_

Reviewer: ```suggestion
>**NOTE:** If you render the Luigi content in a custom container, the container is positioned relatively when you apply your own CSS. Also, set the height of the Luigi custom container either in **px** or **vh**.
```

# Luigi UX features

### Rendering of Luigi application in the DOM

By default, the Luigi content (top navigation, left navigation and content iframed area) are rendered in the body tag of your Luigi Core application. In this case, the Luigi content is taking the whole space from your browser window.

> There is also the possibility to render the Luigi content in any other html container. For example in case you want to add a header or a footer on top of the Luigi content. All you have to do is add the html custom attribute `luigi-app-root` to the html tag where you want to render the Luigi content.

Author: Good point, done.

# Luigi UX features

### Rendering of Luigi application in the DOM

By default, the Luigi content (top navigation, left navigation and content iframed area) are rendered in the body tag of your Luigi Core application. In this case, the Luigi content is taking the whole space from your browser window.

There is also the possibility to render the Luigi content in any other html container. For example in case you want to add a header or a footer on top of the Luigi content. All you have to do is add the html custom attribute `luigi-app-root` to the html tag where you want to render the Luigi content.

> If rendering the Luigi content in a custom container as described above, the height and the width of the Luigi content can be set through the [General Settings](./general-settings.md). 

Reviewer 2: Is this still valid? Guess the user should just specify a height and width on his container?

Author: Good catch! This is not valid anymore. I will remove it.
